src/components/Areas/Areas.scss

```diff
@@ -1,12 +1,17 @@
 .area-card {
-  border: 5px solid black;
-  height: 300px;
+  border: .5em solid black;
+  border-radius: 1em;
+  height: 15em;
   text-decoration: none;
-  width: 300px;
+  margin: auto;
+  padding: 1em;
+  width: 20em;
 }
 
 .Areas-container {
```

Comment:
This class selector's casing should be consistent with everything else you have (eg `areas-container`).

Reply:
Agreed, will get that changed.